devenv/loadtest/render_test.js

```diff
@@ -1,33 +1,58 @@
-import { check, group } from 'k6';
+import { check, fail, group } from 'k6';
 import { createClient } from './modules/client.js';
+import {
+  createTestOrgIfNotExists,
+  upsertTestdataDatasource,
+  upsertTestdataDashboard,
+} from './modules/util.js';
 
 export let options = {
-  noCookiesReset: true
+  noCookiesReset: true,
+  thresholds: { checks: [ { threshold: 'rate=1', abortOnFail: true } ] },
 };
 
 let endpoint = __ENV.URL || 'http://localhost:3000';
 const client = createClient(endpoint);
+const dashboard = JSON.parse(open('fixtures/image-renderer-perf-dashboard.json'));
```

Comment:
This file is missing in your PR but we could also use this one: https://github.com/grafana/grafana-image-renderer/blob/master/devenv/docker/ha/grafana/provisioning/dashboards/general/graph_panel.json (it's a one in use in the current state of the load tests, that are supposed to run with the HA docker setup)

Reply:
Moved that dashboard into the load test directory.